src/dis_tp/structure_functions/f2.py

```diff
@@ -72,7 +72,7 @@ def F2_FO(
                 x,
                 Q,
                 p,
-                h_id,
+                3 if h_id == 5 and nf == 3 else h_id,
```

Comment:
I guess this is only relevant for charm right? This is why 3 and 5 are hardcoded